.enval/enval.tool.yaml

```diff
@@ -0,0 +1,13 @@
+name: enval
```

Comment:
I think we don't need this because it's already in the default tool-specs

Reply:
It is to test the tool override, but taking into account that there is a test checking that may be we can remove it...